cmd/tailscale/cli/web.go

```diff
@@ -437,10 +452,19 @@ func webHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write(buf.Bytes())
 }
 
-func tailscaleUp(ctx context.Context, st *ipnstate.Status, forceReauth bool) (authURL string, retErr error) {
+func tailscaleUp(ctx context.Context, st *ipnstate.Status, postData postedData) (authURL string, retErr error) {
+	if postData.ForceLogout {
+		if err := localClient.Logout(ctx); err == nil {
+			return "", nil
+		} else {
+			return "", fmt.Errorf("Error encountered during logout.")
+		}
```

Comment:
Go style is to early return and outdent the rest. So you can ditch the else.  But then I'd make the first return be the err != nil path, not the err == nil case.

Any reason to omit the detail? Can you fmt.Errorf("Logout error: %w", err) ?

Reply:
Mainly because I'm not certain if the error that eventually comes back will include any sort of implementation detail that would be confusing to the user? I'm happy to replace with your version (and probably also do the same for the login error)

Reply:
If confusing the user means helping a higher chance that we get that info in a bug report and can actually fix it, I'll take it.

I err the other way in control where there are internal messages we don't want to leak out, but this is tailscaled and there aren't really secrets there.

Reply:
Done